lib/config.js

```diff
@@ -45,6 +45,14 @@ const paramsSchema = Joi.object({
       .maxArity(1)
       .optional()
       .default(() => defaultSessionIdGenerator),
+    verificationCookie: Joi.object({
+      sameSite: Joi.string()
+        .valid('Lax', 'Strict', 'None')
+        .optional()
+        .default('Lax'),
```

Comment:
`.default(Joi.ref('..cookie.sameSite'))` (or `Joi.ref('session.cookie.sameSite')` if it's in the root of the config)

Then it doesn't have to be a breaking change

Reply:
Thanks, didn't know about `ref`.